app/views/facility_accounts_reconciliation/_table.html.haml

```diff
@@ -22,5 +21,4 @@
               .order-detail-extra.order-detail-note= order_detail.note
 
         %td= show_actual_total(order_detail)
-        %td= text_field_tag "order_detail[#{order_detail.id}][notes]", begin params[:order_detail][order_detail.id.to_s][:notes] rescue "" end
-  = render partial: "action_row"
+        %td= text_field_tag "order_detail[#{order_detail.id}][reconciled_note]", begin params[:order_detail][order_detail.id.to_s][:reconciled_note] rescue "" end
```

Comment:
Another pre-existing `rescue` in a view! There aren't many left. Same note as https://github.com/tablexi/nucore-open/pull/566#discussion_r61766376